recipes/drmaa/run_test.py

```diff
@@ -0,0 +1,4 @@
+try:
+    import drmaa
```

Comment:
I think you will get a different error if it doesn't point to a real library. If it does point to a real library, but the wrong one, there will be errors about missing symbols and such.

We could check these I suppose. Just not sure if that is where we want to go here.